project.clj

```diff
@@ -1,22 +1,23 @@
-(defproject com.github.shopsmart/clj-foundation "0.9.11"
+(defproject com.github.shopsmart/clj-foundation "0.9.12"
   :description "Common patterns enabling simpler to be easier and harder to be possibler."
   :url "https://github.com/shopsmart/clj-foundation"
 
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
 
-  :plugins [[lein-test-bang-bang "0.2.0"]]
-
-  :profiles {:test {:dependencies [[com.h2database/h2 "1.4.192"]]}
-             :repl {:dependencies [[com.h2database/h2 "1.4.192"]]}}
-
-  :aot :all
+  :plugins [[lein-test-bang-bang "0.2.0"] ; lein test!! - Run each test NS in a separate JRE
+            [lein-ancient "0.6.10"]       ; lein ancient - Check for outdated dependencies
```

Comment:
I'm not sure I would put this in the JAR unless it is used programatically.  Maybe in the documentation note how to add to your lein dev profile.